openaerostruct/geometry/geometry_user_mesh_util.py

```diff
@@ -0,0 +1,661 @@
+""" Utility for quickly generating a multi-section user specificed OAS mesh
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def generateMesh(
+    sections, data, bPanels, cPanels, symmetry=True, controlPoints=False, generatePlots=False, plotOptions={}
+):
+    """
+    Quickly generates user multi-section mesh for analysis based on a few parameters
+
+    Parameters
+    ----------
+    sections : int
+        Integer for number of wing sections specified
+    data : numpy array
+        sectionsx4 array consisting of taper, root chord length, aspect ratio, and leading edge sweep in columns. Each row corresponds to each specified wing section.
+    bPanels: numpy array
+        1 x sections 1-D array consisting of integers that specify the number of spanwise panels corresponding to each wing section
+    cPanels: int
+        Integer for number of chord-wise panels
+    symmetry : boolean
+        Flag set to True if surface is reflected about y=0 plane.
+    controlPoints: boolean
```

Comment:
I honestly not sure. The code is capable of placing control points at the quarter chord which is actually useful for applications where the bound vortex needs to be placed at the leading edge of each panel(i.e vortex lift). I decided to keep the code in and see if anyone else had any feedback or ideas on where it might be useful.